Mage/src/main/java/mage/abilities/common/SpellCastOpponentNoManaSpentTriggeredAbility.java

```diff
@@ -0,0 +1,48 @@
+package mage.abilities.common;
+
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.effects.Effect;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.stack.Spell;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ * @author Susucr
+ */
+public class SpellCastOpponentNoManaSpentTriggeredAbility extends TriggeredAbilityImpl {
+
+    public SpellCastOpponentNoManaSpentTriggeredAbility(Effect effect) {
+        super(Zone.BATTLEFIELD, effect, false);
+        this.setTriggerPhrase("Whenever an opponent casts a spell, if no mana was spent to cast it");
```

Comment:
check text? I thought trigger phrase needed to end with `, `

Reply:
Thanks for the catch. I did indeed miss that.